<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VideoFeed</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Search:</string>
          </property>
          <property name="buddy" >
           <cstring>searchEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="searchEdit" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QTreeView" name="treeView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0" >
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="itemsExpandable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWebView" name="webView" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="url" >
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About" />
    <addaction name="separator" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menu_Feeds" >
    <property name="title" >
     <string>&amp;Feeds</string>
    </property>
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Feeds" />
   <addaction name="menu_Help" />
  </widget>
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About </string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
